packages/weather.rb

```diff
@@ -0,0 +1,46 @@
+require 'package'
+
+class Weather < Package
+  description 'This command-line utility is intended to provide quick access to current weather conditions and forecasts.'
+  homepage 'http://fungi.yuggoth.org/weather/'
+  version '2.3'
+  source_url 'http://fungi.yuggoth.org/weather/src/weather-2.3.tar.xz'
+  source_sha256 '86148d2f1d59867f637f52558cc2a6b3280fac94df55c6e5af0ce37cc190d146'
+
+  depends_on 'python27' unless File.exists? "#{CREW_PREFIX}/bin/python"
+
+  def self.build
+    system "echo 'datadir = #{CREW_PREFIX}/data/weather' >> weatherrc"
+  end
+
+  def self.install
+    system "mkdir -p #{CREW_DEST_PREFIX}/bin"
+    system "mkdir -p #{CREW_DEST_PREFIX}/data/weather"
+    system "mkdir -p #{CREW_DEST_PREFIX}/etc"
+    system "mkdir -p #{CREW_DEST_PREFIX}/man/man1"
+    system "mkdir -p #{CREW_DEST_PREFIX}/man/man5"
+    system "mkdir -p #{CREW_DEST_PREFIX}/$HOME"
+    system "cp weather #{CREW_DEST_PREFIX}/bin"
+    system "cp weather.py #{CREW_DEST_PREFIX}/bin"
+    system "cp airports #{CREW_DEST_PREFIX}/data/weather"
+    system "cp places #{CREW_DEST_PREFIX}/data/weather"
+    system "cp slist #{CREW_DEST_PREFIX}/data/weather"
+    system "cp stations #{CREW_DEST_PREFIX}/data/weather"
+    system "cp zctas #{CREW_DEST_PREFIX}/data/weather"
+    system "cp zlist #{CREW_DEST_PREFIX}/data/weather"
+    system "cp zones #{CREW_DEST_PREFIX}/data/weather"
+    system "cp weatherrc #{CREW_DEST_PREFIX}/etc"
+    system "cp weather.1 #{CREW_DEST_PREFIX}/man/man1"
+    system "cp weatherrc.5 #{CREW_DEST_PREFIX}/man/man5"
+    system "cp weatherrc $HOME/.weatherrc"
+    system "cp weatherrc #{CREW_DEST_PREFIX}/$HOME/.weatherrc"
```

Comment:
Creating `$HOME/.weatherrc` may cause trouble at upgrade.  For example, if you modify `$HOME/.weatherrc` to add your favorite places' shortcut or something, the next upgrade will wipe all those modifications.

If you want to make it automatic, it's better to add something like below in the `postinstall` function.
```
if [ -f $HOME/.wheatherrc ]; then
  # create it
fi
```
Or simple leave messages to copy `/usr/local/etc/wheatherrc` to `$HOME/.wheatherrc`.